tpl/crawler/settings-general.tpl.php

```diff
@@ -28,6 +28,25 @@
 			</td>
 		</tr>
 
+		<tr>
+			<th>
+				<?php $id = Base::O_CRAWLER_SCHEDULE_TIME; ?>
+				<?php $this->title($id); ?>
+			</th>
+			<td>
+				<?php $this->build_input($id); ?>
+				<div class="litespeed-desc">
+					<?php echo __('Specify the crawler running periods.', 'litespeed-cache'); ?>
+					<br />
+					<?php echo sprintf( __('You can add multiple ranges in 24-hour format <code>HH:mm-HH:mm</code> delimited by %s', 'litespeed-cache'), '<code>,</code>'); ?>
```

Comment:
Time format shouldn't be in translatable string.

Reply:
Fixed